src/farcasterd/runtime.rs

```diff
@@ -961,6 +957,16 @@ impl Runtime {
                     )?;
                     Ok(())
                 }
+                BusMsg::P2p(PeerMsg::TakerCommit(TakerCommit { commit, .. })) => {
+                    debug!("{} | Offer already taken or aborted, replying with offer not found to the counterparty", commit.swap_id());
```

Comment:
Maybe log this as info with the offer uuid. I guess it's an interesting info to have as a maker

Reply:
0f8e1fc5195c270c73b402e33e388cc9eba85e4a

Reply:
This is done for taker for now